src/ref/edit.md

```diff
@@ -259,6 +259,44 @@ edit:arg-completer[apt] = [@args]{
 }
 ```
 
+Here is another example of configuring a completer. This time for the `git`
+command. It supports completing some common subcommands and then branch
+names after that:
+
+```elvish
+fn all-git-branches {
+    e:git branch -a --format="%(refname:strip=2)" | eawk [0 1 @rest]{ put $1 }
```

Comment:
Just discovered that the `--format="%(refname:strip=2)"` argument doesn't work on older versions of git (it errors on 2.10.2). Works fine after upgrading to 2.15.0, though. I tried searching through the git release notes and couldn't figure out why.

Reply:
It's fine for education purpose; but please leave a comment here explaining that this assumes a recent version of git.

The `e:` in `e:git` is not needed.